sys/net/gnrc/network_layer/sixlowpan/frag/rbuf.c

```diff
@@ -268,27 +268,27 @@ static rbuf_t *_rbuf_get(const void *src, size_t src_len,
     vtimer_now(&now);
 
     for (unsigned int i = 0; i < RBUF_SIZE; i++) {
-        /* check first if entry already available */
-        if ((rbuf[i].pkt != NULL) && (rbuf[i].pkt->size == size) &&
-            (rbuf[i].tag == tag) && (rbuf[i].src_len == src_len) &&
-            (rbuf[i].dst_len == dst_len) &&
-            (memcmp(rbuf[i].src, src, src_len) == 0) &&
-            (memcmp(rbuf[i].dst, dst, dst_len) == 0)) {
-            DEBUG("6lo rfrag: entry %p (%s, ", (void *)(&rbuf[i]),
-                  gnrc_netif_addr_to_str(l2addr_str, sizeof(l2addr_str),
+        /* check first if entry already available */ 
+	if (rbuf[i].pkt != NULL){
```

Comment:
Whitespace missing and why did you took this out of the rest of the if-statement?